flink-core/src/main/java/org/apache/flink/configuration/Configuration.java

```diff
@@ -349,13 +345,11 @@ public void setLong(ConfigOption<Long> key, long value) {
 	 *        the default value which is returned in case there is no value associated with the given key
 	 * @return the (default) value associated with the given key
 	 */
+	@Deprecated
```

Comment:
I think it would be nicer if for the `@Deprecated` methods, you added the `"@deprecated use the X.Y instead".` to javadoc.